### PR TITLE
PyTorch-Geometric

### PR DESCRIPTION
**Pull request by afraz-ahmed**
For INC1033730

* [x] Assigned to reviewer

PyTorch-Geometric-1.4.3-foss-2019b-Python-3.7.4-PyTorch-1.4.0.eb
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL7-power9
* [ ] Ubuntu16 VM

PyTorch-Geometric-1.4.3-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb
* [ ] EL7-haswell
* [ ] EL7-power9

### CODE REVIEW

File: easybuild/easyconfigs/r/RDFlib/RDFlib-5.0.0-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonBundle'
+
+name = 'RDFlib'
+version = '5.0.0'
```
Comment (edmondac) [inline easybuild/easyconfigs/r/RDFlib/RDFlib-5.0.0-foss-2019b.eb:4]: Do we need 5.0.0? Also https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/r/RDFlib/RDFlib-4.2.2-GCCcore-8.3.0.eb exists upstream

File: easybuild/easyconfigs/r/RDFlib/RDFlib-5.0.0-fosscuda-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonBundle'
+
+name = 'RDFlib'
+version = '5.0.0'
```
Comment (edmondac) [inline easybuild/easyconfigs/r/RDFlib/RDFlib-5.0.0-fosscuda-2019b.eb:4]: Ditto... https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/r/RDFlib/RDFlib-4.2.2-GCCcore-8.3.0.eb

**Comment by afraz-ahmed** [comment]
> Just the question about RDFlib - everything else lgtm

RDFlib changed to 4.2.2. 5.0.0 was not needed; I used it because it was the latest version available.

File: easybuild/easyconfigs/r/RDFlib/RDFlib-5.0.0-foss-2019b.eb (hunk 1 above)
Comment (edmondac) [reply easybuild/easyconfigs/r/RDFlib/RDFlib-5.0.0-foss-2019b.eb]: If 4.2.2 is ok, then let's use upstream's GCCcore version of RDFlib

**Comment by edmondac** [comment]
numba failed on POWER9:
```
extensions sanity check failed for 1 extensions: numba
failing sanity check for 'numba' extension: command "python -c "import numba"" failed; output:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/__init__.py", line 30, in <module>
    from .decorators import (cfunc, generated_jit, jit, njit, stencil, jit_module)
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/decorators.py", line 14, in <module>
    from .targets import registry
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/targets/registry.py", line 5, in <module>
    from . import cpu
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/targets/cpu.py", line 11, in <module>
    from .base import BaseContext, PYOBJECT
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/targets/base.py", line 18, in <module>
    from numba import _dynfunc, _helperlib
ImportError: /rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/_helperlib.cpython-37m-powerpc64le-linux-gnu.so: undefined symbol: pthread_atfork
```

**Comment by edmondac** [comment]
> numba failed on POWER9:
> 
> ```
> extensions sanity check failed for 1 extensions: numba
> failing sanity check for 'numba' extension: command "python -c "import numba"" failed; output:
> Traceback (most recent call last):
>   File "<string>", line 1, in <module>
>   File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/__init__.py", line 30, in <module>
>     from .decorators import (cfunc, generated_jit, jit, njit, stencil, jit_module)
>   File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/decorators.py", line 14, in <module>
>     from .targets import registry
>   File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/targets/registry.py", line 5, in <module>
>     from . import cpu
>   File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/targets/cpu.py", line 11, in <module>
>     from .base import BaseContext, PYOBJECT
>   File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/targets/base.py", line 18, in <module>
>     from numba import _dynfunc, _helperlib
> ImportError: /rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/_helperlib.cpython-37m-powerpc64le-linux-gnu.so: undefined symbol: pthread_atfork
> ```

In the past we've fixed by hacking the easyconfig file like this:
```
import subprocess  # NOQA
arch = subprocess.check_output(['uname', '-m']).strip()
if arch == 'ppc64le':
    # ppc64le has a wrapper libpthread.so that means you can accidentally lose parts of the library
    # see http://ryanarn.blogspot.com/2011/07/curious-case-of-pthreadatfork-on.html
    # This line forces the linker to keep pthread_atfork and pthread_create in the binary
    prebuildopts += ' CFLAGS="$CFLAGS -lpthread -Wl,-u,pthread_atfork -Wl,-u,pthread_create"'
```

**Comment by edmondac** [comment]
Rebuilding LLVM-8.0.1-GCCcore-8.3.0.eb, then on to PyTorch-Geometric-1.4.3-foss-2019b-Python-3.7.4-PyTorch-1.4.0.eb

**Pull request merged**